pkg/controller/certificates/authority/policies.go

```diff
@@ -31,40 +31,76 @@ import (
 type SigningPolicy interface {
 	// not-exporting apply forces signing policy implementations to be internal
 	// to this package.
-	apply(template *x509.Certificate) error
+	apply(template *x509.Certificate, usages []capi.KeyUsage, signerNotAfter time.Time) error
 }
 
 // PermissiveSigningPolicy is the signing policy historically used by the local
 // signer.
 //
 //  * It forwards all SANs from the original signing request.
-//  * It sets allowed usages as configured in the policy.
-//  * It sets NotAfter based on the TTL configured in the policy.
+//  * It sets allowed usages as configured in the policy and based on the requested usages.
+//  * It sets NotBefore based on the Backdate configured in the policy.
+//  * It sets NotAfter based on the TTL and Backdate configured in the policy.
+//    Short lived certificates ignore Backdate and only use TTL.
+//    NotAfter is truncated to the expiration date of the signer.
 //  * It zeros all extensions.
 //  * It sets BasicConstraints to true.
 //  * It sets IsCA to false.
+//  * It validates that the signer has not expired.
 type PermissiveSigningPolicy struct {
-	// TTL is the certificate TTL. It's used to calculate the NotAfter value of
-	// the certificate.
+	// TTL is the certificate TTL. Now and TTL and Backdate are used to calculate the
+	// NotAfter value of the certificate.  Backdate is ignored when TTL is less than Short.
 	TTL time.Duration
-	// Usages are the allowed usages of a certificate.
-	Usages []capi.KeyUsage
+
+	// Backdate and Now are used to calculate the NotBefore value of the certificate.
+	Backdate time.Duration
+
+	// Short is the duration used to determine if the lifetime of a certificate should be considered short.
```

Comment:
might be helpful to repeat the "Short lived certificates ignore Backdate and only use TTL to compute NotAfter." bit in this doc.

Reply:
would it be helpful to say something like this somewhere?
```
All certificates set NotBefore=Now()-Backdate.
Long-lived certificates set NotAfter=Now()+TTL-Backdate.
Short-lived certificates set NotAfter=Now()+TTL.
```

Reply:
See what you think of the updated go docs.